tests/unittests/Foundation/NSRecursiveLockTests.m

```diff
@@ -20,143 +20,6 @@
 #import <thread>
 #import <mutex>
 #import <condition_variable>
-#import <chrono>
-
-#import "NSRecursiveLock+Internal.h"
```

Comment:
There's a tryLock that returns immediately whether the acquisition was successful. The rewrite seems out of scope for this change though - I'll open a separate issue for this?

---
In reply to: [72325371](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72325371) [](ancestors = 72325371)